src/main/java/frc/robot/subsystems/Crane.java

```diff
@@ -76,6 +78,11 @@ public class Crane extends SubsystemBase {
     elevatorPIDF.get().d(),
     new TrapezoidProfile.Constraints(0.0, 0.0) // Dynamically scaled.
   );
+  
+  TunableDouble ks = new TunableDouble("Crane.ks", CraneConstants.kS);
+  TunableDouble kg = new TunableDouble("Crane.kg", CraneConstants.kG);
+  TunableDouble kv = new TunableDouble("Crane.kv", CraneConstants.kV);
```

Comment:
These are `private static final`, and they should be `k[SGV]` for consistency. E.g.

```java
private static final TunableDouble kS = new TunableDouble("Crane.kS", CraneConstants.kS);
```